lmnet/lmnet/networks/object_detection/yolo_v2.py

```diff
@@ -48,7 +48,7 @@ def __init__(
             nms_iou_threshold=0.5,
             nms_max_output_size=100,
             nms_per_class=True,
-            seen_threshold=12800,
+            loss_warmup_steps=100,
```

Comment:
Why do you choose the `100` default value ? Is it from yolov2 paper?

Reply:
@Joeper214 
darknet implementation has `12800` as threshold, 
https://github.com/pjreddie/darknet/blob/f6d861736038da22c9eb0739dca84003c5a5e275/src/region_layer.c#L254
and batch size is `64` when train VOC datasets.
https://github.com/pjreddie/darknet/blob/master/cfg/yolov2-voc.cfg#L6

I recommend the default value of `loss_warmup_steps` is `12800/64 = 200.